crates/subspace-farmer/src/bin/subspace-farmer/commands/bench.rs

```diff
@@ -306,3 +306,202 @@ fn get_size(path: impl AsRef<Path>) -> std::io::Result<u64> {
     }
     Ok(size)
 }
+
+#[cfg(test)]
+#[tokio::test]
+async fn test_dsn_sync() {
```

Comment:
Why is this test in cli bench command of the library and not in the library?

Also should be in a separate module/file, especially considering the size of the test.

Reply:
Because we need benchmarking mocks for it. I can move those to library though with test. Wdut?